pipeline/mutate/mutator_id_token.go

```diff
@@ -208,5 +208,28 @@ func (a *MutatorIDToken) Config(config json.RawMessage) (*CredentialsIDTokenConf
 		c.TTL = "15m"
 	}
 
+	cost := int64(c.Cache.MaxCost)
+	if cost == 0 {
+		cost = 1 << 25
+	}
+
+	if a.tokenCache == nil || a.tokenCache.MaxCost() != cost {
+		cache, err := ristretto.NewCache(&ristretto.Config[string, *idTokenCacheContainer]{
+			NumCounters: cost * 10,
```

Comment:
So counters should actually be 10 * number of items. Since the cost no longer == number of items, it's not trivial to set this. But this is probably too high?

Reply:
Yeah, being able to estimate the number of JWTs that we can fit in the given max size relates to the average size of JWTs. But it's gonna be tough to get an "average JWT size" when the payload is configurable 😅 

What would be a better guess in your opinion?
I pushed this for now, which would leave a bit less extra space through the number of counters.
```suggestion
			NumCounters: cost * 4,
```